CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.1.0](https://github.com/johmsalas/text-case.nvim/compare/v1.0.0...v1.1.0) (2023-12-20)
```

Comment:
@peterfication should we release for every new change? :thinking:

Reply:
I guess it will be also CI/CD now :thought_balloon:

Reply:
Yeah, I was asking myself this as well ...

Reply:
Let's do it for this one! I will start observing how other plugins do
You can approve and merge